dusty/commands/disk.py

```diff
@@ -1,28 +1,59 @@
 import logging
+import os
 
 import docker
 
+from .. import constants
 from ..log import log_to_client
+from ..path import dir_modified_time, set_mac_user_ownership
 from ..systems.docker.cleanup import remove_exited_dusty_containers, remove_images
-from ..systems.virtualbox import get_docker_vm_disk_info, ensure_docker_vm_is_started
+from ..systems.virtualbox import (get_docker_vm_disk_info, ensure_docker_vm_is_started,
+                                  set_read_persist_permission, set_write_persist_permission)
+from ..systems.rsync import sync_local_path_to_vm, sync_local_path_from_vm
 
 def cleanup_inactive_containers():
-    log_to_client("Ensuring virtualbox vm is running")
+    log_to_client("Ensuring virtualbox VM is running")
     ensure_docker_vm_is_started()
     log_to_client("Cleaning up exited containers:")
     containers = remove_exited_dusty_containers()
     log_to_client("Done cleaning {} containers".format(len(containers)))
 
 def cleanup_images():
-    log_to_client("Ensuring virtualbox vm is running")
+    log_to_client("Ensuring virtualbox VM is running")
     ensure_docker_vm_is_started()
     log_to_client("Cleaning up docker images without containers:")
     images = remove_images()
     log_to_client("Done removing {} images".format(len(images)))
 
 def inspect_vm_disk():
-    log_to_client("Ensuring virtualbox vm is running")
+    log_to_client("Ensuring virtualbox VM is running")
     ensure_docker_vm_is_started()
     log_to_client("Boot2Docker VM Disk Usage:")
     log_to_client(get_docker_vm_disk_info())
 
+def _full_destination_dir(destination_path):
+    return os.path.join(destination_path, constants.LOCAL_BACKUP_DIR)
+
+def _ensure_backup_dir_exists(destination_path):
+    if not os.path.exists(destination_path):
+        os.makedirs(destination_path)
+
+def backup(path):
+    destination_path = _full_destination_dir(path)
```

Comment:
I could go either way on this.  If we add `dusty-backup` to the folder path, then when a user does `dusty disk backup ~/`, for example, he'll get a folder called `~/dusty-backup` containing the stuff, instead of a bunch like `~/coreMongo`, `~/coreRedis`, etc